dashboard/pipeline.js

```diff
@@ -43,6 +43,12 @@ let stream = pipeline
     })
   })
 
+  // Share activity metrics with Access Watch Hub and get updates
+  .map(log => {
```

Comment:
Shouldn't we modify instead the hub augmentation API to pass the whole log and take care of everything there? This is kind of awkward to have two entry points to the hub plugin with different sets of assumptions. If the hub needs to know the log format then let's take advantage of it. And we wouldn't have to call the hub plugin twice from the pipeline.

Reply:
@benfle I'm open with the idea, I will look into it.